miasm2/arch/x86/arch.py

```diff
@@ -549,7 +549,10 @@ def to_string(self, loc_db=None):
         if self.additional_info.g1.value & 2:
             if getattr(self.additional_info.prefixed, 'default', "") != "\xF2":
                 o = "REPNE %s" % o
-        if self.additional_info.g1.value & 4:
+        if self.additional_info.g1.value & 8:
```

Comment:
I'd have to add a variable to keep the growing order. It causes https://travis-ci.org/cea-sec/miasm/jobs/489927927#L1304 otherwise(it resolves REP LODSD as REP REPE LODSD due to it).

Reply:
Hoo!!!
Correct, sorry for the bad comment !